tests/gis_tests/test_geoforms.py

```diff
@@ -111,7 +111,8 @@ class PointForm(forms.Form):
 
         self.assertInHTML(
             '<textarea id="id_pt1" class="vSerializedField required" cols="150"'
-            ' rows="10" name="pt1">POINT (7.3 44)</textarea>',
+            ' rows="10" name="pt1">{&quot;coordinates&quot;: [7.3, 44.0],'
```

Comment:
Actually `{&quot;coordinates&quot;: [7.3, 44.0], &quot;type&quot;: &quot;Point&quot;}` renders in a non-deterministic order, hence the failure.

@claudep, sorry I noticed you linked to the wrong build in the original PR but I didn't point it out.